admin/index.html

```diff
@@ -30,6 +30,18 @@
       button {
         margin-top: 0px !important;
       }
+      .css-g42qkj-ToolbarSubSectionFirst .css-16pehfe-StyledDropdownButton-button-default-caret-caretDown-DropdownButton-noOverflow-PublishedToolbarButton-publishedButton  {
```

Comment:
What is `g42qkj`? It looks like some kind of generated hash - is this going to change if we update netlify, for example?

Reply:
@okkays I've been playing around with how to do this without these classes and there's not a great way that I could find. I'm thinking we go with this in the short term and I will create a PR against the Netlify repo to see if we can get a more permanent fix implemented.

Reply:
sgtm!